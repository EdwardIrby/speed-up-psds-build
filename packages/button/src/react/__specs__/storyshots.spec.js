import React from 'react'
import initStoryshots from '@storybook/addon-storyshots'

jest.mock('../../../.storybook/center-decorator.js', () => {
  const CenterDecorator = props => (
    <div data-storybook-center-decorator {...props} />
  )
  return CenterDecorator
})

initStoryshots()
