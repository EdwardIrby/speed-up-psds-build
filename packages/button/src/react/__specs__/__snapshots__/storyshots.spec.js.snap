// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Button / appearance flat 1`] = `
<button
  data-css-mtq3v8=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / appearance primary 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / appearance secondary 1`] = `
<button
  data-css-1fl0ces=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / appearance stroke 1`] = `
<button
  data-css-1q6fyq5=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / as link default 1`] = `
<a
  data-css-1srt63x=""
  disabled={false}
  href="https://duckduckgo.com"
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click as link
  </span>
</a>
`;

exports[`Storyshots Button / as link with icon 1`] = `
<a
  data-css-t7n654=""
  disabled={false}
  href="https://duckduckgo.com"
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="pencil icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M5.276 15.862L5 19l3.14-.275 8.085-8.086-2.863-2.863-8.086 8.086zM17.64 9.225l.767-.767a2 2 0 00.592-1.431c0-.546-.208-1.048-.593-1.433a1.996 1.996 0 00-1.43-.591c-.545 0-1.048.208-1.433.593l-.766.766 2.863 2.863zm-.665-6.222c1.076 0 2.086.418 2.845 1.177A3.997 3.997 0 0121 7.027a4 4 0 01-1.179 2.846L9.411 20.282a.995.995 0 01-.464.263l-4.705.427a1.002 1.002 0 01-1.213-1.213l.427-4.704a.998.998 0 01.263-.464L14.13 4.18a3.997 3.997 0 012.845-1.179z"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Click as link
  </span>
</a>
`;

exports[`Storyshots Button / disabled all 1`] = `
<div
  style={
    Object {
      "display": "grid",
      "gap": "20px",
      "gridTemplateColumns": "1fr 1fr",
    }
  }
>
  <button
    data-css-1srt63x=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      primary
    </span>
  </button>
  <button
    data-css-1xtru9q=""
    disabled={true}
    onClick={[Function]}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      primary disabled
    </span>
  </button>
  <button
    data-css-1fl0ces=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      secondary
    </span>
  </button>
  <button
    data-css-1xtru9q=""
    disabled={true}
    onClick={[Function]}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      secondary disabled
    </span>
  </button>
  <button
    data-css-1q6fyq5=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      stroke
    </span>
  </button>
  <button
    data-css-1e8144b=""
    disabled={true}
    onClick={[Function]}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      stroke disabled
    </span>
  </button>
  <button
    data-css-mtq3v8=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      flat
    </span>
  </button>
  <button
    data-css-j7ib5k=""
    disabled={true}
    onClick={[Function]}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      flat disabled
    </span>
  </button>
</div>
`;

exports[`Storyshots Button / disabled with icon 1`] = `
<button
  data-css-t9q1pk=""
  disabled={true}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="pencil icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M5.276 15.862L5 19l3.14-.275 8.085-8.086-2.863-2.863-8.086 8.086zM17.64 9.225l.767-.767a2 2 0 00.592-1.431c0-.546-.208-1.048-.593-1.433a1.996 1.996 0 00-1.43-.591c-.545 0-1.048.208-1.433.593l-.766.766 2.863 2.863zm-.665-6.222c1.076 0 2.086.418 2.845 1.177A3.997 3.997 0 0121 7.027a4 4 0 01-1.179 2.846L9.411 20.282a.995.995 0 01-.464.263l-4.705.427a1.002 1.002 0 01-1.213-1.213l.427-4.704a.998.998 0 01.263-.464L14.13 4.18a3.997 3.997 0 012.845-1.179z"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Disabled
  </span>
</button>
`;

exports[`Storyshots Button / icon flat 1`] = `
<button
  data-css-17xa4jo=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon large 1`] = `
<button
  data-css-f4nzhh=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon left 1`] = `
<button
  data-css-t7n654=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon lone flat large 1`] = `
<button
  data-css-1cyn5sp=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone flat medium 1`] = `
<button
  data-css-ch3vk9=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone flat small 1`] = `
<button
  data-css-1hcldyp=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone flat xSmall 1`] = `
<button
  data-css-lp1ejc=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-12f5sv2=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone primary large 1`] = `
<button
  data-css-1d0jjqh=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone primary medium 1`] = `
<button
  data-css-ym649h=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone primary small 1`] = `
<button
  data-css-18ap1id=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone primary xSmall 1`] = `
<button
  data-css-cj5s2z=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-12f5sv2=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone secondary large 1`] = `
<button
  data-css-18pflo1=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone secondary medium 1`] = `
<button
  data-css-1rgey3m=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone secondary small 1`] = `
<button
  data-css-fu22rr=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone secondary xSmall 1`] = `
<button
  data-css-1ia90sa=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-12f5sv2=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone stroke large 1`] = `
<button
  data-css-ism3iy=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone stroke medium 1`] = `
<button
  data-css-1si5h8m=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone stroke small 1`] = `
<button
  data-css-4w2p0c=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon lone stroke xSmall 1`] = `
<button
  data-css-71ikm9=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-12f5sv2=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / icon medium 1`] = `
<button
  data-css-t7n654=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon primary 1`] = `
<button
  data-css-t7n654=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon right 1`] = `
<button
  data-css-klviz3=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-8b1vtw=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon secondary 1`] = `
<button
  data-css-d8pbsb=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon small 1`] = `
<button
  data-css-1kfn5ea=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon stroke 1`] = `
<button
  data-css-vmxpv1=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / icon vertically aligned rows 1`] = `
<div>
  <button
    data-css-t7n654=""
    disabled={false}
    style={Object {}}
  >
    <div
      data-css-1mgd019=""
    >
      <div
        data-css-otejxm=""
      >
        <svg
          aria-label="check icon"
          role="img"
          viewBox="0 0 24 24"
        >
          <path
            d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
          />
        </svg>
      </div>
    </div>
    <span
      data-css-15b13by=""
    >
      With icon
    </span>
  </button>
  <button
    data-css-1srt63x=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      Without icon
    </span>
  </button>
</div>
`;

exports[`Storyshots Button / icon xSmall 1`] = `
<button
  data-css-1hi09ns=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-12f5sv2=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    With Icon
  </span>
</button>
`;

exports[`Storyshots Button / in row same vertical height 1`] = `
<div>
  <div
    style={
      Object {
        "display": "grid",
        "gap": "8px",
        "gridTemplateColumns": "auto auto auto auto",
        "marginBottom": "8px",
      }
    }
  >
    <button
      data-css-uh7ewg=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-js91uc=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1n1un6q=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-lwnd5m=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
  </div>
  <div
    style={
      Object {
        "display": "grid",
        "gap": "8px",
        "gridTemplateColumns": "auto auto auto auto",
        "marginBottom": "8px",
      }
    }
  >
    <button
      data-css-12jrzwh=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1bx03j2=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-110xkhb=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1rlm1mh=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
  </div>
  <div
    style={
      Object {
        "display": "grid",
        "gap": "8px",
        "gridTemplateColumns": "auto auto auto auto",
        "marginBottom": "8px",
      }
    }
  >
    <button
      data-css-1srt63x=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1fl0ces=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1q6fyq5=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-mtq3v8=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
  </div>
  <div
    style={
      Object {
        "display": "grid",
        "gap": "8px",
        "gridTemplateColumns": "auto auto auto auto",
        "marginBottom": "8px",
      }
    }
  >
    <button
      data-css-1g9x2u4=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1c873lm=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1noii5v=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
    <button
      data-css-1d7sx6d=""
      disabled={false}
      style={Object {}}
    >
      <span
        data-css-15b13by=""
      >
        button
      </span>
    </button>
  </div>
</div>
`;

exports[`Storyshots Button / loading flat 1`] = `
<button
  data-css-pbgf66=""
  disabled={true}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-1ec065u=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading large 1`] = `
<button
  data-css-1vyuucj=""
  disabled={true}
  onClick={[Function]}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-10bxom6=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading lone icon 1`] = `
<button
  data-css-1xawxh6=""
  disabled={true}
  style={Object {}}
>
  <div
    data-css-y80vay=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-10bxom6=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  />
</button>
`;

exports[`Storyshots Button / loading medium 1`] = `
<button
  data-css-19xn5cc=""
  disabled={true}
  onClick={[Function]}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-10bxom6=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading no icon, hidden text 1`] = `
<button
  data-css-1g9x2u4=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Text doesnt show when loading if no icon
  </span>
</button>
`;

exports[`Storyshots Button / loading primary 1`] = `
<button
  data-css-1vyuucj=""
  disabled={true}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-10bxom6=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading secondary 1`] = `
<button
  data-css-1h92z2u=""
  disabled={true}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-9pakmv=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading small 1`] = `
<button
  data-css-1qdzo6w=""
  disabled={true}
  onClick={[Function]}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-10bxom6=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading stroke 1`] = `
<button
  data-css-1oclh3v=""
  disabled={true}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <span
        data-css-1jut5p0=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / loading xSmall 1`] = `
<button
  data-css-1newt3g=""
  disabled={true}
  onClick={[Function]}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-12f5sv2=""
    >
      <span
        data-css-10bxom6=""
      />
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Loading...
  </span>
</button>
`;

exports[`Storyshots Button / override styles with className 1`] = `
<button
  className={
    Object {
      "data-css-jxpnuu": "",
    }
  }
  data-css-t7n654=""
  disabled={false}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Green Button
  </span>
</button>
`;

exports[`Storyshots Button / override styles with style 1`] = `
<button
  data-css-t7n654=""
  disabled={false}
  style={
    Object {
      "background": "red",
    }
  }
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Red Button
  </span>
</button>
`;

exports[`Storyshots Button / props pass through anchor supports download 1`] = `
<a
  data-css-1srt63x=""
  disabled={false}
  download={true}
  href="/somewhere"
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Link with download
  </span>
</a>
`;

exports[`Storyshots Button / props pass through aria-expanded 1`] = `
<button
  aria-expanded={true}
  data-css-1srt63x=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    aria-expanded
  </span>
</button>
`;

exports[`Storyshots Button / props pass through button doesnt support download 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Button with download
  </span>
</button>
`;

exports[`Storyshots Button / props pass through data-something 1`] = `
<button
  data-css-1srt63x=""
  data-something="wow"
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Custom data attributes
  </span>
</button>
`;

exports[`Storyshots Button / props pass through not supported 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  onMouseOver={[Function]}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Should not mouseover
  </span>
</button>
`;

exports[`Storyshots Button / props pass through role 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  role="link"
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Role Link
  </span>
</button>
`;

exports[`Storyshots Button / props pass through title 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  style={Object {}}
  title="My caption"
>
  <span
    data-css-15b13by=""
  >
    With title
  </span>
</button>
`;

exports[`Storyshots Button / size large 1`] = `
<button
  data-css-1g9x2u4=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / size medium 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / size small 1`] = `
<button
  data-css-12jrzwh=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / size xSmall 1`] = `
<button
  data-css-uh7ewg=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Click me
  </span>
</button>
`;

exports[`Storyshots Button / theme nested 1`] = `
Array [
  <button
    data-css-mtq3v8=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      dark
    </span>
  </button>,
  <button
    data-css-qguorc=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      light
    </span>
  </button>,
  <button
    data-css-mtq3v8=""
    disabled={false}
    style={Object {}}
  >
    <span
      data-css-15b13by=""
    >
      dark
    </span>
  </button>,
]
`;

exports[`Storyshots Button / with onClick clicks once 1`] = `
<button
  data-css-t7n654=""
  disabled={false}
  onClick={[Function]}
  style={Object {}}
>
  <div
    data-css-1mgd019=""
  >
    <div
      data-css-otejxm=""
    >
      <svg
        aria-label="check icon"
        role="img"
        viewBox="0 0 24 24"
      >
        <path
          d="M9.59 14.58l-3.17-3.17L5 12.82l4.59 4.59 10-10L18.18 6"
        />
      </svg>
    </div>
  </div>
  <span
    data-css-15b13by=""
  >
    Clicks once
  </span>
</button>
`;

exports[`Storyshots Button / with ref ref to handle focus 1`] = `
<button
  data-css-1srt63x=""
  disabled={false}
  style={Object {}}
>
  <span
    data-css-15b13by=""
  >
    Should be focused
  </span>
</button>
`;
